.github/workflows/main.yml

```diff
@@ -88,12 +90,18 @@ jobs:
         echo "BUILD_LIBSETJMP=no" >> $GITHUB_ENV
       if: matrix.clang_version == '10.0.0'
 
+    - name: Enable PIC build for new LLVM
+      shell: bash
+      run: |
+        echo "MAKE_TARGETS=default libc_so" >> $GITHUB_ENV
+      if: matrix.enable_pic
+
     - name: Build libc
       shell: bash
       run: |
-        make -j4 TARGET_TRIPLE=wasm32-wasi
-        make -j4 TARGET_TRIPLE=wasm32-wasip1
-        make -j4 TARGET_TRIPLE=wasm32-wasip2 WASI_SNAPSHOT=p2
```

Comment:
What happened to `WASI_SNAPSHOT=p2` here?

Reply:
Oops, it was my mistake.